.travis.yml

```diff
@@ -33,10 +33,24 @@ jobs:
     - stage: Build test
       language: minimal
       before_script:
-        - LATEST_AZCOPY="$(curl -sSLo- https://api.github.com/repos/Azure/azure-storage-azcopy/releases | jq -r '.[0].tag_name' | sed 's/^v//g')"
-        - test -n "$LATEST_AZCOPY"
-        - export LATEST_AZCOPY
+        - if [ "master" = "$SOURCE_BRANCH" ]; then
+            AZCOPY_VERSION=$(curl -sLo- https://api.github.com/repos/Azure/azure-storage-azcopy/releases | grep tag_name | head -n 1 | awk -F'"' '{print $4}' | sed 's/^v//g');
+          else
+            AZCOPY_VERSION="$SOURCE_BRANCH";
+          fi
+        - export AZCOPY_VERSION
```

Comment:
Doesn't look like we need to export it, but I believe that we can print (`echo`) it to help we debug.